05-classification.Rmd

```diff
@@ -5,8 +5,9 @@ knitr::opts_chunk$set(message = FALSE, warning = FALSE, echo = TRUE)
 ```
 
 ## Overview 
-Up until this point, we have focused solely on descriptive and exploratory
```

Comment:
"about data" at the end of the first two sentences reads a bit akward to me when I read it out loud, is there another way we could say this? Perhaps:

"In previous chapters, we focused solely on descriptive and exploratory data analysis questions." 

I suggest this, as we call these questions types of data analysis questions in chapter 1.

Reply:
agree!